modules/contacts/controller.js

```diff
@@ -1,28 +1,29 @@
 const { isValidObjectId } = require("mongoose");
-const Joi = require("joi");
 const contactsService = require("./service");
 
-const schema = Joi.object({
-  name: Joi.string().min(3).max(30),
-  email: Joi.string().email({
-    minDomainSegments: 2,
-    tlds: { allow: ["com", "net"] },
-  }),
-  phone: Joi.string(),
-});
-
 const listContacts = async (req, res, next) => {
-  const contacts = await contactsService.getAll();
+  let contacts = await contactsService.getAll();
+  const { page, limit, favorite } = req.query;
+
+  if (favorite) {
+    contacts = contacts.filter((contact) => contact.favorite.toString() === favorite);
```

Comment:
tutaj dwie rzeczy:
1. Czym jest `contact.favorite`, ze musisz je zmieniac w String? 🤔 (wspolczesniej jest tez zrobic `String(contact.favorite)`.
2. Mysle, ze spokojnie mozesz zrobic tutaj tak (przy zalozeniu, ze `contact.favorite` to juz String (jesli nie, to go "strignij" :) )
`...filter(({favorite: fav}) => fav === favorite)` (troche czytelniej)